### ISSUE TITLE
Protocol error: FORMAT DEBEZIUM forbids additional columns, but got ["_rw_kafka_timestamp"]

### ISSUE THREAD

**Issue by gaozuo** [risingwavelabs/risingwave#18222]
### Describe the bug

CREATE SOURCE IF NOT EXISTS cdc_hr_waf_core_entity
WITH (
    connector = 'kafka',
    --'debezium-avro-confluent.schema-registry.url' = 'http://kafka-stream-registry-cp-schema-registry.kafka-cluster:8081',
    --'format' = 'debezium-avro-confluent',
    properties.bootstrap.server = 'kafka-stream-kafka-bootstrap.kafka-cluster:9092',
    --'properties.group.id' = 'flink-cdc',
    scan.startup.mode = 'earliest',
    topic = 'HR.HROUTPUT.WAF_CORE_ENTITY'
)
FORMAT debezium ENCODE AVRO (
  schema.registry = 'http://kafka-stream-registry-cp-schema-registry.kafka-cluster:8081'
)

### Error message/log

```text
org.jkiss.dbeaver.model.sql.DBSQLException: SQL 错误 [XX000]: ERROR: Failed to execute the statement

Caused by:
  Protocol error: FORMAT DEBEZIUM forbids additional columns, but got ["_rw_kafka_timestamp"]


	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)

	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:614)

	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$2(SQLQueryJob.java:505)

	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:194)

	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:524)

	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:976)

	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:4155)

	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)

	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:194)

	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)

	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5152)

	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:115)

	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

Caused by: org.postgresql.util.PSQLException: ERROR: Failed to execute the statement

Caused by:
  Protocol error: FORMAT DEBEZIUM forbids additional columns, but got ["_rw_kafka_timestamp"]


	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2725)

	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2412)

	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:371)

	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:502)

	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:419)

	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:341)

	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:326)

	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:302)

	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:297)

	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)

	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)

	... 12 more
```

**Comment by gaozuo** [comment]
use "CREATE SOURCE" just fine.

**Comment by TiansuYu** [comment]
I am currently experiencing the same type of error with consuming DEBEZIUM AVRO from Kafka:
```
psycopg2.errors.InternalError_: Failed to run the query

Caused by:
  Protocol error: FORMAT DEBEZIUM forbids additional columns, but got ["_rw_kafka_timestamp", "_rw_kafka_partition", "_rw_kafka_offset"]
```
The CREATE SOURCE statement is as follows:
```sql
CREATE SOURCE my_source
WITH (
    connector = 'kafka',
    topic = 'my_topic',
    properties.bootstrap.servers = '{{ env["KAFKA_BOOTSTRAP_SERVERS"] }}',
    properties.group.id = 'my_group',
    scan.startup.mode = 'earliest-offset',
    properties.security.protocol = 'SASL_SSL',
    properties.sasl.mechanism = 'PLAIN',
    properties.sasl.username = '{{ env["KAFKA_USERNAME"] }}',
    properties.sasl.password = '{{ env["KAFKA_PASSWORD"] }}'
)
FORMAT DEBEZIUM ENCODE AVRO (
    schema.registry = '{{ env["SCHEMA_REGISTRY_URL"] }}',
    ignore.key = 'true' -- with or without this 
);
```
Could you please shed some light here, in how to debug this?

**Comment by james-johnston-thumbtack** [comment]
I am also experiencing the same problem as @TiansuYu with the exact same error message on RisingWave v2.2.  My SQL statement is very similar to theirs.

**Comment by TiansuYu** [comment]
I got someone answered this on slack: for debezium, you need to use CREATE TABLE (not SOURCE), this fixed this issue

**Comment by hzxa21** [comment]
> I got someone answered this on slack: for debezium, you need to use CREATE TABLE (not SOURCE), this fixed this issue

The error message is confusing in previous versions. It has been fixed in #20582 which is included in version `2.3`
